app/config/config.yml

```
@@ -21,7 +21,11 @@ services:
     PrestaShopBundle\Controller\:
         resource: "%kernel.root_dir%/../src/PrestaShopBundle/Controller/*"
         exclude: "%kernel.root_dir%/../src/PrestaShopBundle/Controller/Api"
-        tags: ['prestashop.core.controllers']
+        tags:
+          - 'controller.service_arguments'
+          - !php/const PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController::PRESTASHOP_CORE_CONTROLLERS
```

```suggestion
          - !php/const PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController::PRESTASHOP_CORE_CONTROLLERS_TAG
```
So we understand what it stands for (it's not a service name for example)

---

Oh and you have conflict on this file by the way, and there's an extra line you can remove maybe

---

Done in https://github.com/PrestaShop/PrestaShop/pull/18811/commits/e8ed3b04116ea196210b185283e373fba6cc998a